tests/eval_hub/scripts/test_kube_rbac_proxy.sh

```
@@ -0,0 +1,321 @@
+        echo "${response_body:0:500}..." >> "$OUTPUT_FILE"
+    else
+        echo "$response_body" >> "$OUTPUT_FILE"
+    fi
+    echo '```' >> "$OUTPUT_FILE"
+    echo "" >> "$OUTPUT_FILE"
+
+    # Get evalhub logs if request reached the backend
+    if [ "$actual_status" = "200" ] || [ "$actual_status" = "202" ] || [ "$actual_status" = "409" ]; then
+        echo "**evalhub Logs (showing received headers):**" >> "$OUTPUT_FILE"
+        echo '```json' >> "$OUTPUT_FILE"
+        local pod=$(get_evalhub_pod)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)